src/services/apiUrl.js

```diff
@@ -1,9 +1,11 @@
-const api = process.env.REACT_APP_BASE_URL;
+// const api = process.env.REACT_APP_BASE_URL;
 // const api = process.env.REACT_APP_DEV_URL;
```

Comment:
remove comments

Reply:
I need that for setting dev or prod url.